doc/source/whatsnew/v1.0.0.rst

```diff
@@ -135,7 +135,9 @@ Categorical
 
 - Added test to assert the :func:`fillna` raises the correct ValueError message when the value isn't a value from categories (:issue:`13628`)
 - Bug in :meth:`Categorical.astype` where ``NaN`` values were handled incorrectly when casting to int (:issue:`28406`)
--
+- Using date accessors on a categorical dtyped series of datetimes was not returning a Series (or DataFrame) of the
```

Comment:
I'm having a hard time following this. Is the "not returning a Series / DataFrame" the same issue as the "skipping duplicates?"

If they're distinct issue, break it into multiple lines.

Reply:
@TomAugspurger the only issue is that it's acting on the unique categories rather than on the values. I've tried rephrasing this more clearly